lib/timescaledb/database/quoting.rb

```diff
@@ -0,0 +1,12 @@
+module Timescaledb
+  class Database
+    module Quoting
+      # Quotes given value
+      #
+      # @return [String] The given value between quotes
+      def quote(value)
+        "'#{value}'"
```

Comment:
In theory, table names shouldn't allow using of quotes, but this quoting value is not prepared to escape inner quotes. We could delegate it from external libs if available or implement it if necessary:


```ruby
def quote(value)
   if defined? ActiveRecord
     ...
   elsif defined? Sequel
      ...
   else
      "'#{value.tr("'", '\\')}"
   end
end

Reply:
Good catch 🚀. I pushed new changes; I used rails quote implementation; as it is that simple I wouldn't add ActiveRecord and Sequel conditions here. Added a few specs to be sure the method is working properly. Let me know what you think about not adding ActiveRecord and Sequel conditions.